app/models/SicStore.scala

```diff
@@ -37,7 +40,13 @@ case class SearchResults(query: String,
 
 case class Sector(code: String,
                   name: String,
-                  count: Int)
+                  nameCy: String,
+                  count: Int) {
+  def nameLabel(implicit messages: Messages, appConfig: AppConfig): String = messages.lang.code match {
```

Comment:
You need to update the views that use the sector name to use the nameLabel def instead

Reply:
👍